src/main/java/org/opentripplanner/api/common/RoutingResource.java

```diff
@@ -908,19 +908,29 @@ protected RoutingRequest buildRequest() throws ParameterException {
     }
 
     /**
-     * Take a string in the format agency:id or agency:id:1:2:3:4.
-     * TODO Improve Javadoc. What does this even mean? Why are there so many colons and numbers?
-     * Convert to a Map from trip --> set of int.
+     * Create a banned trips map.
+     *
+     * @param banned A string with a multi-part format.
+     *               The first part consists of TRIP_ID,TRIP_ID (multiple trip IDs separated by commas)
+     *               The TRIP_ID has various IDs separated by colons. There must be at least two colon-separated values,
+     *               otherwise the TRIP_ID is skipped. The first value indicates teh agency ID string. The second value
+     *               indicates the trip ID string within the context of the agency. Any further values must be integers
+     *               representing stop sequences within the trip that are banned. If only the agency ID and trip ID are
+     *               provided, the all stops within the trip will be banned.
      */
     private HashMap<FeedScopedId, BannedStopSet> makeBannedTripMap(String banned) {
```

Comment:
This javadoc is super helpful. It might be good to enhance the `RoutingResource#bannedTrips` doc as well. And, just for a quick visual cue in this method's javadoc, add an example string.

Reply:
Extra documentation added in 6adec27faa2e744f94cd91e4ae4d337a5c25c7c7